modules/widgets.php

```diff
@@ -41,14 +41,23 @@ function jetpack_load_widgets() {
 function jetpack_widgets_loaded() {
 	Jetpack::enable_module_configurable( __FILE__ );
 	Jetpack::module_configuration_load( __FILE__, 'jetpack_widgets_configuration_load' );
+	add_filter( 'jetpack_module_configuration_url_widgets', 'jetpack_widgets_configuration_url' );
 }
 
 function jetpack_widgets_configuration_load() {
 	wp_safe_redirect( admin_url( 'widgets.php' ) );
 	exit;
 }
 
-
+/**
+ * Overrides default configuration url
+ *
+ * @uses admin_url
+ * @return string module settings URL
+ */
+function jetpack_widgets_configuration_url() {
+	return admin_url( 'widgets.php' );
```

Comment:
Maybe we could redirect to the customizer, since it's a newer interface to interact with widgets?